examples/js/loaders/GLTFLoader.js

```diff
@@ -2028,43 +2023,39 @@ THREE.GLTFLoader = ( function () {
 
 	/**
 	 * Specification: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#cameras
+	 * @param {number} cameraIndex
+	 * @return {Promise<THREE.Camera>}
 	 */
-	GLTFParser.prototype.loadCameras = function () {
-
-		var json = this.json;
+	GLTFParser.prototype.loadCamera = function ( cameraIndex ) {
 
-		return _each( json.cameras, function ( camera ) {
+		var camera;
+		var cameraDef = this.json.cameras[ cameraIndex ];
+		var params = cameraDef[ cameraDef.type ];
 
-			var _camera;
+		if ( !params ) {
```

Comment:
Very minor thing tho, you need an space after `!` as the coding style.

`if ( ! params ) {`